src/zkbob/ZkBobPool.sol

```diff
@@ -34,6 +34,7 @@ abstract contract ZkBobPool is IZkBobPool, EIP1967Admin, Ownable, Parameters, Ex
 
     uint256 internal constant MAX_POOL_ID = 0xffffff;
     bytes4 internal constant MESSAGE_PREFIX_COMMON_V1 = 0x00000000;
+    bytes4 internal constant MESSAGE_PREFIX_COMMON_V2 = 0x00000100;
```

Comment:
For DD we use `0x0001`,  do you think it makes more sense to use `0x0002` instead of `0x0100` here?

Reply:
I have the same question :) It indeed looks a bit weird.

My initial idea was to use one byte as the operation type (transact, dd, something else) and one byte for the version within that operation type. So for example the third version of dd can be encoded as 0x0201. Since the last byte was already occupied I used the only available third byte. What do you think?